inc/tagitem.class.php

```diff
@@ -342,11 +343,9 @@ static function updateItem(CommonDBTM $item, $options = array()) {
       $tag      = new PluginTagTag();
       $tag_item = new self();
 
-      // untokenize values and create new ones
-      $tag_values = [];
-      if (!empty($item->input["_plugin_tag_tag_values"])) {
```

Comment:
Are you sure about the check removal?

Reply:
I changed the input from a text input to a select multiple, so values are now passed as array instead of a chain with coma separated values. Exploding is not need anymore.

Reply:
It is not about the explode, but about the "empty". Removing this may lend to undefined index, I do not know in this specific case.

Reply:
OK, I was missing not thinking about possible foreach issue.

Reply:
By the way, a isset test is made at begining of the function.